battlebots/database/access.py

```diff
@@ -37,7 +37,8 @@ def remove_bot(user, botname):
                         % (user.nickname, botname, code_dir))
         pass
 
-    bot = session.query(Bot).filter_by(user=user, name=botname).one_or_none()
```

Comment:
@Silox: I just reread your first comments. Why wouldn't there be an open session? If we chose (2), than either the ranker or the web server should have opened one. Otherwise this code shouldn't be called in the first place. Right?